test/mason/EXECENV

```diff
@@ -2,7 +2,8 @@
 
 import os
 import sys
+import subprocess
```

Comment:
subprocess import is not needed

Reply:
I changed it to a bash script and many to links. Didn't see problems until I paratested (which is apparently the environment that leaves chpl not in PATH).